keanu-python/keanu/vertex/const.py

```diff
@@ -14,17 +14,20 @@ def Const(t) -> Vertex:
         ctor = __infer_const_ctor_from_ndarray(val)
     elif isinstance(t, primitive_types):
         ctor = __infer_const_ctor_from_scalar(t)
-        val = np.array([[t]])
+        val = t
     else:
         raise NotImplementedError("Argument t must be either an ndarray or an instance of numbers.Number. Was given {} instead".format(type(t)))
 
     return ctor(Tensor(val))
 
 def __infer_const_ctor_from_ndarray(ndarray):
-    if len(ndarray) == 0:
-        raise ValueError("Cannot infer type because the ndarray is empty")
+    if isinstance(ndarray, np.ndarray):
```

Comment:
`numpy_types`

Reply:
Why do we even have to check this, if `__infer_const_ctor_from_ndarray` is only called at loc 10 and 14?